tests/FlatBuffers.Test.Swift/Tests/FlatBuffers.Test.SwiftTests/FlatBuffersVectorsTests.swift

```diff
@@ -32,7 +32,7 @@ final class FlatBuffersVectors: XCTestCase {
         b.finish(offset: o)
         XCTAssertEqual(b.sizedByteArray, [4, 0, 0, 0, 0, 0, 0, 0])
     }
-
+    
```

Comment:
this diff has all these space changes everywhere, is that intentional?

Reply:
@aardappel this was intentional, we are using the default formatting for xcode now

Reply:
Xcode has spaces on empty lines? Most editors will undo this, so that would be weird.